uvtools/plot.py

```diff
@@ -723,3 +723,185 @@ def plot_diff_uv(uvd1, uvd2, pol=None, check_metadata=True, bins=50):
         fig.colorbar(cax)
 
     return fig
+
+def plot_diff_1d(uvd1, uvd2, antpairpol, plot_type="both", 
+                 check_metadata=True, dimension=None):
+    # TODO: docstring
+    """Produce plots of visibility differences along a single axis.
+
+    Parameters
+    ----------
+    uvd1, uvd2 : pyuvdata.UVData
+        UVData objects which store visibilities to be differenced and their
+        associated metadata. They should have the same number of frequencies,
+        same baselines, and same times as each other.
+
+    antpairpol : tuple
+        Tuple specifying which baseline and polarization to use to compare 
+        visibility waterfalls. See pyuvdata.UVData.get_data method docstring 
+        for information on accepted tuples.
+    
+    plot_type : str, optional
+        A string identifying which quantities to plot. Accepted values are 
+        as follows:
+            - base
```

Comment:
It's hard to choose good names for these settings, but `normal` and `fourier` or similar seems a bit clearer to me.

Reply:
I agree; I'll update it accordingly.